owncloudApp/src/main/java/com/owncloud/android/AppExecutors.kt

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2017 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.owncloud.android
+
+import android.os.Handler
+import android.os.Looper
+
+import java.util.concurrent.Executor
+import java.util.concurrent.Executors
+
+/**
+ * Global executor pools for the whole application.
+ *
+ * Grouping tasks like this avoids the effects of task starvation (e.g. disk reads don't wait behind
+ * webservice requests).
+ */
+open class AppExecutors(
+    private val diskIO: Executor,
+    private val networkIO: Executor,
+    private val mainThread: Executor
+) {
+    constructor() : this(
+        Executors.newSingleThreadExecutor(),
+        Executors.newFixedThreadPool(3),
+        MainThreadExecutor()
+    )
+
+    fun diskIO(): Executor {
```

Comment:
try always to make functions in one line for these cases:

`fun diskIO(): Executor = diskIO`

Apply for all functions in this file